gulpfile.babel.ts

```diff
@@ -6,30 +6,19 @@ const packageJson = JSON.parse(
   fs.readFileSync(__dirname + '/package.json', 'utf8')
 );
 
-import _ from 'lodash';
 import gulp from 'gulp';
 import destAtomic from 'gulp-dest-atomic';
-import browserify from 'browserify';
-import watchify from 'watchify';
-import source from 'vinyl-source-stream';
-import streamify from 'gulp-streamify';
-import gulpif from 'gulp-if';
-import terser from 'gulp-terser';
-import sourcemaps from 'gulp-sourcemaps';
+import webpack from 'webpack';
 import stdio from 'stdio';
-import gutil from 'gulp-util';
 import extReloader from './live/extReloader';
-import Kefir from 'kefir';
 import fg from 'fast-glob';
 import exec from './src/build/exec';
-import babelify from 'babelify';
-import lazyPipe from 'lazypipe';
-import concat from 'gulp-concat';
-import addsrc from 'gulp-add-src';
 
 const sdkFilename = 'inboxsdk.js';
 
 const args = stdio.getopt({
+  // --watch is a node flag that doesn't seem to work after Node 18.12 with gulp.
```

Comment:
Node 18.15 seems to recognize `--watch` and then fail. `-w` makes it through to the underlying gulp script though.